annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FieldSetterNameGenerator.java

```diff
@@ -0,0 +1,21 @@
+package com.exactpro.epfast.annotation.processing;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class FieldSetterNameGenerator {
+    public static String generateClassName(String fastTypeName) throws NoSuchAlgorithmException {
```

Comment:
If we pass both fastPackage and fastName we can create hash from both of them and place all files in one Java package.

Or we may keep it as is and encode just type name without hashing it first